cmd/demo/node.go

```diff
@@ -488,3 +514,16 @@ func Setup() {
 	}
 	backend = b
 }
+
+//etherToWei converts amount in "ether" (represented as decimal) to "wei" (represented as integer)
```

Comment:
Here it is called `decimal` and in the next function `float`.

Reply:
Addressed in this commit - e10c196.

Reply:
Addressed in this commit - e10c196.